tlf/test_common.go

```diff
@@ -16,6 +16,17 @@ func FakeID(b byte, public bool) ID {
 	return ID{bytes}
 }
 
+// FakeIDRandomOrBust creates a fake public or private TLF ID from the given
+// byte, and fill the rest with empty bytes.
```

Comment:
IMO this doesn't need to be in the `kbfs` repo. And I don't like setting the byte. We can always compare with the randomly generated `TlfID`.